### PR TITLE
Document installation onto NVMe feature

### PR DESCRIPTION
**Pull request by agners**
This feature is supported with the latest Home Assistant OS Installer for Yellow (see https://github.com/NabuCasa/yellow-buildroot/releases/tag/yellow-installer-20230405 and https://github.com/home-assistant/version/pull/285).

### CODE REVIEW

File: src/procedures/procedures.11tydata.yaml [hunk 1]
```diff
@@ -204,6 +204,9 @@ proceduresInstallingHAOS:
       content: |
         <ul>
           <li>If you are not using PoE, connect the power supply to Home&nbsp;Assistant&nbsp;Yellow&nbsp;Kit.</li>
+          <li class="info">Advanced: Home Assistant OS is installed onto the eMMC if available on your CM4. If you
```
Comment (c0ffeeca7) [inline src/procedures/procedures.11tydata.yaml:207]:
```suggestion
          <li class="info"><b>Advanced</b>: Installing the Home Assistant OS onto the NVMe SSD: </br> By default, Home Assistant OS is installed onto the eMMC. Unless you have a CM4 Lite, which does not have an eMMC. If you 
```

**Review by c0ffeeca7 — APPROVED** [review]

**Review by c0ffeeca7 — CHANGES_REQUESTED** [review]
maybe we could give a bit more context there

**Comment by agners** [comment]
> maybe we could give a bit more context there

What do you mean exactly?

The context should be largely given: OS installation while using the NVMe (as this is what this guide is about).

**Comment by c0ffeeca7** [comment]
> > maybe we could give a bit more context there
> 
> What do you mean exactly?
> 
> The context should be largely given: OS installation while using the NVMe (as this is what this guide is about).

I was talking about the sentence I rephrased. That's all :-)

**Review by c0ffeeca7 — APPROVED** [review]

**Pull request merged**